scripts/roles.json

```diff
@@ -4680,7 +4682,6 @@
     "requiredOwnedElements": [],
     "props": [
       "aria-expanded",
-      "aria-modal",
```

Comment:
I'm going nuts trying to figure out why the associated modules for the roles aren't changing when I run `node scripts/breakUpAriaJSON.js`. Turns out there's another `aria-modal` on line 4704 :/